### DISCUSSION TITLE
[Help]: Create Sandbox button not working

### DISCUSSION THREAD

**Discussion by BrScoti** [davidjerleke/embla-carousel#886] [Help] [answered]
### Summary

I'm doing everything requested for the creation, but the button does not allow clicking and generating the component; the predefined ones are working as they should.

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment, marked as answer]
@BrScoti where is this happening? On the generator page?

**Comment by BrScoti** [comment]
That's right, but from what I noticed, the problem was actually a CSS error. The div PageGrid__TableOfContentsWrapper is over the button, so the action was disabled. Removing the class makes it work again.
![Screenshot from 2024-05-21 16-25-51](https://github.com/davidjerleke/embla-carousel/assets/21362482/44e527bf-1226-42f1-908b-11945eed808b)

**Reply by davidjerleke** [reply]
@BrScoti I've released a fix for this bug now. Would you mind testing it and see if's resolved on your side?

- [https://www.embla-carousel.com/examples/generator/](https://www.embla-carousel.com/examples/generator/)

**Reply by TalMoshel1** [reply]
Hi David,

I'm having the same issue now. 

![carousel generator](https://github.com/user-attachments/assets/1167885c-7403-41f6-bee5-4259d7dafdf5)

**Reply by davidjerleke** [reply]
@TalMoshel1 you need to choose framework before the button is enabled.

**Reply by TalMoshel1** [reply]
Thank you
